app/controllers/medications_controller.rb

```diff
@@ -73,18 +67,12 @@ def update
   # DELETE /medications/1.json
   def destroy
     @medication.destroy
-    respond_to do |format|
-      format.html { redirect_to medications_path }
-      format.json { head :no_content }
-    end
+    redirect_to_path(medications)
```

Comment:
This one should be `medications_path`

Can you write a test that proves this doesn't work and then fix it?

Reply:
Oh shoot yeah. Can you address this in another PR @camillevilla?

Reply:
whoops! will do @HashNotAdam @julianguyen